lib/scrape.js

```diff
@@ -184,6 +185,32 @@ async function findOrganization({
   return null
 }
 
+async function fetchSEtag(wikidata_id) {
+  const url = wdk.getEntities({
+    ids: [wikidata_id],
+    props: ['claims'],
+  })
+  const page = await fetch(url)
+  let data = await page.json()
+  data = data.entities[wikidata_id]
+  //P1482 is Stack Exchange tag
+  if ('P1482' in data.claims) {
+    const url = wdk.simplify.claim(data.claims.P1482[0])
+    const response = await fetch(url)
+    const txt = await response.text()
+    const html = cheerio.load(txt)
```

Comment:
Is there any API for SE? We could use that instead of dirty scrapping.

Reply:
There is, but I don't think it's very useful since there is no direct way to get the favicon, and besides, it'll need a key etc.